Source/WebCore/dom/PopoverData.h

```diff
@@ -28,6 +28,7 @@
 #include "Element.h"
 #include "HTMLElement.h"
 #include "HTMLFormControlElement.h"
+#include "ToggleEvent.h"
```

Comment:
We should avoid including this by forward declaring `ToggleEventTask` and having `ensureToggleEventTask` and the `PopoverData` constructor and destructor not be inlined. Can still use `default`, but in the `.cpp` file.

Doing this as we go helps prevent compile time slowdowns that accrue over time when we unnecessarily add header includes to headers.